extensions/agent/src/dialogs/jobDialog.ts

```diff
@@ -229,7 +229,7 @@ export class JobDialog extends AgentDialog<JobData>  {
 						this.StepsTable_FailureColumnString
 					],
 					data: data,
-					height: 650
+					height: 500
```

Comment:
is there a way to get the height that doesn't require hard-coding?  Shouldn't this be based on the size of the window to actually work well in different resolutions?

Reply:
Unfortunately, there's no way to get the size of the window from the extension. A possible solution is to also support heights and widths in percentages. I'll try implementing that and see if it solves these problems without breaking anything.